site-packages/integralstor_utils/disks.py

```diff
@@ -591,25 +575,43 @@ def get_disk_info_all():
         return return_dict, None
 
 
+def is_mounted(disk_name):
+    """Given a disk name like sda, returns true if its mounted"""
+    try:
+        cmd = "cat /proc/mounts | grep %s" % disk_name
+        output, err = command.get_command_output(
+            cmd, shell=True, check_rc=False)
+        if err:
+            raise Exception(err)
+        elif output == []:
+            is_mounted = False
+        else:
+            is_mounted = True
+    except Exception, e:
+        return None, "Unable to check for device %s: %s" % (disk_name, str(e))
+    else:
+        return is_mounted, None
+
+
 def get_file_system_by_name(disk_name):
     """get file system type by providing disk name or partition name"""
```

Comment:
Since you are returning fs_type from the else, define it outside the try and initialize it to None or something else.